modules/oauth-customizing-the-oauth-server-URL.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * authentication/configuring-internal-oauth.adoc
+
+[id="customizing-the-oauth-server-url_{context}"]
+= Customizing the internal OAuth server URL
+
+You can customize the internal OAuth server URL by setting the custom host name and TLS certificate in the `spec.componentRoutes` field of the cluster `Ingress` configuration.
+
+.Prerequisites
+
+* You have logged in to the cluster as a user with administrative privileges.
+* You have created a secret in the `openshift-config` namespace containing the TLS certificate and key. This is required if the domain for the custom host name suffix does not match the cluster domain suffix. The secret is optional if the suffix matches.
++
+[TIP]
+====
+You can create a TLS secret by using the `oc create secret tls` command.
```

Comment:
@slaskawi I added this in response to your line "The best way to ensure that is to use the TLS secret.". Let me know if this is along the right lines of what you meant.

Reply:
You got that perfectly! Thanks!